dind/build-cluster.sh

```diff
@@ -49,13 +53,15 @@ cp ${KUBE_ROOT}/bazel-bin/test/e2e/e2e.test ${CLUSTER_DIR}
 cat ${KUBE_ROOT}/bazel-out/stable-status.txt | grep STABLE_BUILD_SCM_REVISION | awk '{print $2}' > ${CLUSTER_DIR}/source_version
 cat ${KUBE_ROOT}/bazel-out/stable-status.txt | grep STABLE_DOCKER_TAG | awk '{print $2}' > ${CLUSTER_DIR}/docker_version
 
-# Get systemd wrapper (needed until we have systemd run start.sh at startup).
+# Get systemd wrapper (needed until we put cluster-up into a systemd target).
 cp ${TOOL_ROOT}/init-wrapper.sh ${CLUSTER_DIR}
 # Get the cluster-up script.
 cp ${TOOL_ROOT}/start.sh ${CLUSTER_DIR}
 # Get the test execution script.
 cp ${TOOL_ROOT}/dind-test.sh ${CLUSTER_DIR}
 
+cp ${TOOL_ROOT}/../bazel-bin/dind/cmd/cluster-up/linux_amd64_stripped/cluster-up ${CLUSTER_DIR}
```

Comment:
this path may not work for cross compile, you might consider building this tool with `pure = on` (no cgo) and explicitly for linux amd64 to get a stable path

Reply:
I can definitely do that. But I'm not super familiar with bazel cross compile, how would the path be perturbed if built for that target explicitly?

I'm asking for a friend, who is also building a lot of other bazel binaries (e.g., ginkgo).